### ISSUE TITLE
netdicom

### ISSUE THREAD

**Issue by ASVBPREAUBV** [suyashkumar/dicom#41]
Did you get rid of the Implementations in github.com/grailbio/go-netdicom?
Would like to switch from grailbio, but dont want use both.
Maybe i can implement a separate dcm server depending on your implementation.

**Comment by suyashkumar** [comment]
hm if that library depended on certain APIs in the grailbio dicom parser, those have probably been changed or removed, as there was a lot of extensive refactoring done on our end (which has led to a lot of improvements imo, but did require cutting some things out). There is some support for querying a DICOM dataset here: https://github.com/suyashkumar/dicom/blob/master/query/queryretrieve.go

Will look into this more, but I'd love to explore some of the netdicom stuff!

**Comment by siddharthab** [comment]
Hello, just a quick note to say we are getting ready to archive https://github.com/grailbio/go-dicom and https://github.com/grailbio/go-netdicom. For go-dicom, we will point new users to this project, and for go-netdicom, we will point users to this issue to track any progress on DIMSE that is compatible with this project.

**Comment by antibios** [comment]
I've attempted to move go-netdicom to support the suyashkumar DICOM implementation.  Have a look here https://github.com/antibios/go-netdicom

**Comment by sebheitzmann** [comment]
@antibios is it possible to open issue in your project, to contribute ?

**Comment by antibios** [comment]
@sebheitzmann yes please I would love to get some feedback on the implementation.

**Comment by suyashkumar** [comment]
@antibios thanks for putting something together! I'll take a look later this week as well :).
